Library/Formula/neon.rb

```diff
@@ -14,12 +11,12 @@ class Neon < Formula
     sha1 "2273ce25ead7be557b71101303cfda284364cd7b" => :mountain_lion
   end
 
-  keg_only :provided_by_osx
+  keg_only :provided_pre_mountain_lion
```

Comment:
I haven't actually checked Mountain Lion or Mavericks, but it doesn't seem to exist on my Yosemite system and it disappeared from [Apple Open Source](https://www.opensource.apple.com/) between 10.7.5 and 10.8.0.